ts/packages/cli/src/commands/export.ts

```diff
@@ -1,10 +1,21 @@
 import { cwd } from 'process';
 import { spawnSync } from 'child_process';
 import { Flags, ux } from '@oclif/core';
-import _ from 'lodash';
+import _, { kebabCase } from 'lodash';
```

Comment:
Instead of using lodash, let's keep using `change-case` for this feature. For kebab case, you can use `paramCase` from `change-case`. They recently released a new major version where they renamed paramCase to kebabCase but I don't want you to have to deal with potential breaking changes.

Reply:
Cool. thanks